app/assets/stylesheets/components/buttons.scss

```
@@ -48,6 +48,7 @@
   cursor: pointer;
   transition: all var(--transition-props);
   border: var(--border-width) solid;
+  overflow-wrap: break-word;
```

```suggestion
  overflow-wrap: normal;
```

I think this should be 'normal'? I checked locally and it still fixes the issue - according to [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) this makes sure:

> Lines may only break at normal word break points (such as a space between two words).

Which I think is what we'd want for our buttons?